gemato/openpgp.py

```diff
@@ -19,6 +19,8 @@
 GNUPG = os.environ.get('GNUPG', 'gpg')
 GNUPGCONF = os.environ.get('GNUPGCONF', 'gpgconf')
 
+ENVIRON_WHITELIST = ['GNUPGHOME', 'http_proxy', 'https_proxy']
```

Comment:
Well, you don't want to allow GNUPGHOME from env, that would override our tempdir.

Reply:
Yes indeed.